manifests/init.pp

```diff
@@ -12,6 +12,8 @@
 #
 # === Advanced parameters:
 #
+# $foreman_fqdn::                       FQDN of the foreman server (usually equal to parent_fqdn)
```

Comment:
Maybe parent_fqdn needs a longer description or at least mention that it's the parent pulp node